app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+
+    km = 0
+
+    def __init__(self, kilometer):
+        self.km = kilometer
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        return Distance(kilometer=self.km + other.km) \
```

Comment:
You can just use this construction: `return Distance(self.km + other.km) \`... and then you won`t need line separation in this block.

Reply:
I can't. Because if I want to use somthing like this:

> dst = Distance(10)
> print(dst + 10)

I will get the error AttributeError: 'int' object has no attribute 'km'